src/Intermediary.tsx

```diff
@@ -40,33 +106,35 @@ export const Intermediary: React.FunctionComponent = () => {
   }, []);
 
   return (
-    <Card
-      variant="outlined"
-      sx={{
-        bgcolor: "background.paper",
-        boxShadow: 1,
-        borderRadius: 2,
-        p: 2,
-        minWidth: 300,
-      }}
-    >
-      <img
-        src={logo}
-        className="logo"
-        alt="SCBridge-Wallet Logo"
-        style={{ height: "25vh" }}
-      />
-      <h2>Intermediary</h2>
-      <AddressIcon
-        address={
-          // @ts-expect-error
-          import.meta.env.VITE_IRENE_ADDRESS as `0x${string}`
-        }
-      />
-      <div className="card">
-        <p>Owner balance: {ownerBalance}</p>
-        <p>Intermediary balance: {intermediaryBalance}</p>
-      </div>
-    </Card>
+    <>
+      <Stack
+        direction="row"
+        justifyContent="left"
+        alignItems="center"
+        spacing={2}
+      >
+        <Tooltip title={props.client.ownerAddress} placement="top">
+          <Avatar
+            src={blo(props.client.ownerAddress as `0x${string}`)}
+            sx={{ width: 24, height: 24 }}
+          />
+        </Tooltip>
+        <Typography>Owner balance: {ownerBalance}</Typography>
+      </Stack>
+      <Stack
+        direction="row"
+        justifyContent="left"
+        alignItems="center"
+        spacing={2}
```

Comment:
nit: there doesn't seem to be any veritical spacing on my machine

Reply:
Yeah this is basically trial and error from me at the moment!